cmd/kubelet/app/options/options.go

```diff
@@ -139,7 +139,7 @@ type KubeletFlags struct {
 func NewKubeletFlags() *KubeletFlags {
 	return &KubeletFlags{
 		ContainerRuntimeOptions: *NewContainerRuntimeOptions(),
-		CertDirectory:           "/var/lib/kubelet/pki",
+		CertDirectory:           filepath.Join(filepath.Clean(defaultRootDir), "pki"),
```

Comment:
You can drop the explicit `filepath.Clean`, `filepath.Join` cleans up automatically.